Sprint-1/interpret/to-pounds.js

```diff
@@ -1,27 +1,36 @@
 const penceString = "399p";
 
 const penceStringWithoutTrailingP = penceString.substring(
-  0,
-  penceString.length - 1
-);
+  0, // extracts a portion of the string here starting from index 0
```

Comment:
Seems to have some typo in this comment.
Some of the other comments are not precise enough.